src/Nethermind/Nethermind.Db.Test/DbMetricsUpdaterTests.cs

```diff
@@ -29,18 +29,27 @@ public void ProcessCompactionStats_AllDataExist()
-            Assert.That(Metrics.DbStats[("TestDb", "Level1FilesCompacted")], Is.EqualTo(2));
-            Assert.That(Metrics.DbStats[("TestDb", "Level2Files")], Is.EqualTo(3));
-            Assert.That(Metrics.DbStats[("TestDb", "Level2FilesCompacted")], Is.EqualTo(1));
-            Assert.That(Metrics.DbStats[("TestDb", "IntervalCompactionGBWrite")], Is.EqualTo(10));
-            Assert.That(Metrics.DbStats[("TestDb", "IntervalCompactionMBPerSecWrite")], Is.EqualTo(2));
-            Assert.That(Metrics.DbStats[("TestDb", "IntervalCompactionGBRead")], Is.EqualTo(123));
-            Assert.That(Metrics.DbStats[("TestDb", "IntervalCompactionMBPerSecRead")], Is.EqualTo(0));
-            Assert.That(Metrics.DbStats[("TestDb", "IntervalCompactionSeconds")], Is.EqualTo(111));
+            Assert.That(Metrics.DbStats.Count, Is.EqualTo(5));
+            // Level    Files   Size     Score Read(GB)  Rn(GB) Rnp1(GB) Write(GB) Wnew(GB) Moved(GB) W-Amp Rd(MB/s) Wr(MB/s) Comp(sec) CompMergeCPU(sec) Comp(cnt) Avg(sec) KeyIn KeyDrop Rblob(GB) Wblob(GB)
+            // ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+            // L0     94/0    1.22 GB   0.1      0.2     0.3      0.4      0.5     0.6       0.7   0.8      0.9     1.0    1.1            1.2      1.3    1.4       0      0       0.0       0.0
+
+            Assert.That(Metrics.DbCompactionStats[("TestDb", 0, "files")], Is.EqualTo(94));
+            Assert.That(Metrics.DbCompactionStats[("TestDb", 0, "files_compacting")], Is.EqualTo(0));
+            Assert.That(Metrics.DbCompactionStats[("TestDb", 0, "score")], Is.EqualTo(0.1));
+            Assert.That(Metrics.DbCompactionStats[("TestDb", 0, "size")], Is.EqualTo(1309965025.28));
+            Assert.That(Metrics.DbCompactionStats[("TestDb", 0, "read")], Is.EqualTo(0.2));
+            Assert.That(Metrics.DbCompactionStats[("TestDb", 0, "rn")], Is.EqualTo(0.3));
+            Assert.That(Metrics.DbCompactionStats[("TestDb", 0, "rnp1")], Is.EqualTo(0.4));
+            Assert.That(Metrics.DbCompactionStats[("TestDb", 0, "write")], Is.EqualTo(0.5));
+            Assert.That(Metrics.DbCompactionStats[("TestDb", 0, "wnew")], Is.EqualTo(0.6));
+            Assert.That(Metrics.DbCompactionStats[("TestDb", 0, "moved")], Is.EqualTo(0.7));
+            Assert.That(Metrics.DbCompactionStats[("TestDb", 0, "wamp")], Is.EqualTo(0.8));
+            Assert.That(Metrics.DbCompactionStats[("TestDb", 0, "rd")], Is.EqualTo(0.9));
+            Assert.That(Metrics.DbCompactionStats[("TestDb", 0, "wr")], Is.EqualTo(1.0));
+            Assert.That(Metrics.DbCompactionStats[("TestDb", 0, "comp_sec")], Is.EqualTo(1.1));
+            Assert.That(Metrics.DbCompactionStats[("TestDb", 0, "comp_merge_cpu_sec")], Is.EqualTo(1.2));
+            Assert.That(Metrics.DbCompactionStats[("TestDb", 0, "comp_total")], Is.EqualTo(1.3));
```

Comment:
This would look so much better as separate TestCases.

Reply:
Nah, its fine here.